src/middlewares/index.js

```diff
@@ -0,0 +1,9 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+// @flow
+
+// This file just reexports all middlewares for an easier access.
+
+export * from './versioning';
```

Comment:
I kind of want to do this pattern on the profiler front-end... The only real downside is that it broke jump to definition for me previously (not sure if it works now or not).

Reply:
ah, I don't know :) my own "jump to definition" is grep...